### ISSUE TITLE
[Feature] setMembers / setAttribute for members

### ISSUE THREAD

**Issue by MattesDe** [DirectoryTree/LdapRecord#436]
Hi,
we are migrating from Adldap2 to LdapRecord. It looks like the setMembers method for groups is no longer available.
Especially for large groups with thousands of members it was quite easy to update the members this way.
It is not possible to use `$group->setAttribute('member', $DnList);` (this is what setMembers does in Adldap2) for large groups (> 1500 members).
This is what an existing large group look like:
`"member" => [],
  "member;range=0-1499" => array:1500 [
      0 => "CN=...`

If we add more than 1500 members via setAttribute, we get `ldap_modify_batch(): Batch Modify: Already exists`.
Is there any chance to get the feature back?
Thanks a lot.

**Comment by MattesDe** [comment]
Okay, after digging deeper into this we found out the following:
It looks like if the AD server has a limit for attributes, the original "member" attribute is not filled when the limit is exceeded. Only the attribute "member;range=0-1499" is filled.
If now the "member" attribute is filled, there is no "ldap_mod_replace" in save(), instead an " ldap_mod_add" is executed. This leads to the error "Batch Modify: Already exists".
We fixed that by using "updateAttribute()" for the members update, which always uses a "ldap_mod_replace".
